### PR TITLE
[test-only] add issue tag to failed tests

### LINKED ISSUE #5125: A User can get information of another user with Graph API
### Description
With `Graph API` a user can get information of another user with username of another user.

### Steps to reproduce
1. Create `user1` and `user2`

#### Curl command to get user information
```
curl -X GET -vk -u user1:123456 https://host.docker.internal:9200/graph/v1.0/users/user2 
```

### Expected Behaviour
`user1` should not be albe to get the information of another user `user2`

### Actual Behaviour with the curl request
```
* SSL connection using TLSv1.3 / TLS_AES_128_GCM_SHA256
* ALPN, server did not agree to a protocol
* Server certificate:
*  subject: O=Acme Corp; CN=OCIS
*  start date: Nov 25 08:23:00 2022 GMT
*  expire date: Nov 25 08:23:00 2023 GMT
*  issuer: O=Acme Corp; CN=OCIS
*  SSL certificate verify result: unable to get local issuer certificate (20), continuing anyway.
* Server auth using Basic with user 'user1'
} [5 bytes data]
> GET /graph/v1.0/users/user2 HTTP/1.1
> Host: host.docker.internal:9200
> Authorization: Basic dXNlcjE6MTIzNDU2
> User-Agent: curl/7.68.0
> Accept: */*
> 
{ [5 bytes data]
* TLSv1.3 (IN), TLS handshake, Newsession Ticket (4):
{ [130 bytes data]
* Mark bundle as not supporting multiuse
< HTTP/1.1 200 OK
< Content-Length: 135
< Content-Type: application/json; charset=utf-8
< Date: Fri, 25 Nov 2022 08:34:44 GMT
< X-Graph-Version: 2.0.0-rc.1+b4e727b60
< 
{ [135 bytes data]
100   135  100   135    0     0   3000      0 --:--:-- --:--:-- --:--:--  3000
* Connection #0 to host host.docker.internal left intact
{
  "displayName": "user2 gurung",
  "id": "205ccfc7-3c37-46aa-9339-d7dc2ad5e362",
  "mail": "user2@gmail.com",
  "onPremisesSamAccountName": "user2"
}

```

### Environment
`oCIS` latest docker image

### PR DESCRIPTION
**Pull request by ScharfViktor**
I've cleaned up a bit
- deleted closed issue from expected failures file 
- added issue tag to tests

I analyzed the number of failed tests:

| Passed |Failed |% of failed tests | 
|-------|-------|-------|
| 1549 | 277 | 17,9% |

If we talk about the new api, then
| Passed |Failed |% of failed tests | 
|-------|-------|-------|
| 860 | 122 | 14,2% |

if we fix #5125 #5938 #5939 we remove 56 failed tests from expected failures list

### CODE REVIEW

**Comment by ownclouders** [comment]
E2E tests failed: https://drone.owncloud.com/owncloud/ocis/20983/68/1
<details><summary>:boom: To see the trace, please open the link in the console ...</summary>

<p>


#### npx playwright show-trace https://cache.owncloud.com/public/owncloud/ocis/20983/tracing/file-alice-2023-3-30-10-09-17.zip 


</p></details>

**Review by individual-it — APPROVED** [review]

**Comment by ScharfViktor** [comment]
e2e flaky, if you don't mind I bump weCommit here

File: tests/acceptance/expected-failures-localAPI-on-OCIS-storage.md [hunk 1]
```diff
@@ -62,7 +62,7 @@ The expected failures in this file are from features in the owncloud/ocis repo.
 #### [Share lists deleted user as 'user'](https://github.com/owncloud/ocis/issues/903)
 - [apiGraph/deleteGroup.feature:68](https://github.com/owncloud/ocis/blob/master/tests/acceptance/features/apiGraph/deleteGroup.feature#L68)
 
-#### [Renaming the group with = character in it doesn't work](https://github.com/owncloud/ocis/issues/5969)
+#### [Updating group displayName request seems OK but group is not being renamed](https://github.com/owncloud/ocis/issues/5977)
```
Comment (saw-jan) [inline tests/acceptance/expected-failures-localAPI-on-OCIS-storage.md:65]: Should be https://github.com/owncloud/ocis/issues/5969? was recently changed in PR https://github.com/owncloud/ocis/pull/5971
```suggestion
#### [Renaming the group with = character in it doesn't work](https://github.com/owncloud/ocis/issues/5969)
```

File: tests/acceptance/features/apiArchiver/downloadByPath.feature [hunk 2]
```diff
@@ -65,12 +65,12 @@ Feature: download multiple resources bundled into an archive
       | my_data/textfile2.txt       | some data  |
       | more_data/an_other_file.txt | more data  |
 
-
+
```
Comment (saw-jan) [inline tests/acceptance/features/apiArchiver/downloadByPath.feature:68]:
```suggestion

```

File: tests/acceptance/expected-failures-localAPI-on-OCIS-storage.md (hunk 1 above)
Comment (ScharfViktor) [reply tests/acceptance/expected-failures-localAPI-on-OCIS-storage.md:65]: 5969 is closed issue. I created new and link tests

File: tests/acceptance/features/apiArchiver/downloadByPath.feature (hunk 2 above)
Comment (ScharfViktor) [reply tests/acceptance/features/apiArchiver/downloadByPath.feature:68]: I change it in next PR. I don't want to run CI again

**Pull request merged**